internal/validate/api_resources_test.go

```
@@ -0,0 +1,269 @@
+		t.Fatalf("unexpected error: %v", err)
+	}
+
+	if index["v1"]["Namespace"].Resource.Namespaced {
+		t.Fatal("expected Namespace to be cluster-scoped (namespaced=false)")
+	}
+	if !index["v1"]["Pod"].Resource.Namespaced {
+		t.Fatal("expected Pod to be namespaced (namespaced=true)")
+	}
+}
+
+func writeAPIResourcesFile(t *testing.T, dir string) string {
```

golangci-lint (2.11.4)

[error] func writeAPIResourcesFile is unused

(unused)